test/cluster/test_hints.py

```
@@ -500,3 +604,797 @@ async def migration_reached_streaming():
+    # Re-enable hinted handoff.
+    await update_hh_enabled_via_http_api(manager, s1, "true")
+    await wait_until_hints_are_sent_from(manager, [s1], expected_rows)
+
+    await manager.server_stop_gracefully(s1.server_id)
+
+    cql = await manager.get_cql_exclusive(s2)
+    read_stmt = cql.prepare("SELECT pk FROM ks.t WHERE pk >= ? AND pk < ? ALLOW FILTERING")
+    read_stmt.consistency_level = ConsistencyLevel.ONE
+
+    batch_size = 1000
+    for batch_idx in range(0, expected_rows // batch_size):
```

Ruff (0.16.3)

[warning] Unnecessary `start` argument in `range`

Remove `start` argument

(PIE808)